Ansible/Ansible.php

```diff
@@ -15,6 +15,7 @@
 use Asm\Ansible\Command\AnsiblePlaybook;
 use Asm\Ansible\Command\AnsiblePlaybookInterface;
 use Asm\Ansible\Exception\CommandException;
+use Symfony\Component\Process\Process;
```

Comment:
@xabbuh After a bit of testing, I'm not able to reproduce it in a fresh project. This is my fault (sorry about that!), I need to figure out what's specifically causing me issues. We can safely roll back the additional use statement.